sources/packages/web/src/components/fa-application/Section.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <div class="p-component p-mx-4">
     <div class="p-grid">
```

Comment:
Looks like this component existed before this PR and you modified it. But can you add a comment somewhere to this file to say in what scenarios should some page use this component

Reply:
Added some comments.